multicluster/controllers/multicluster/stale_controller.go

```diff
@@ -39,18 +39,17 @@ import (
 )
 
 const (
-	LeaderCluster = "leader"
-	MemberCluster = "member"
-
-	memberClusterAnnounceStaleTime = 5 * time.Minute
+	LeaderCluster                  = "leader"
+	MemberCluster                  = "member"
+	memberClusterAnnounceStaleTime = 24 * time.Hour
 )
 
 // StaleResCleanupController will clean up ServiceImport, MC Service, ACNP, ClusterInfoImport and LabelIdentity
 // resources if no corresponding ResourceImports in the leader cluster and remove stale ResourceExports
 // in the leader cluster if no corresponding ServiceExport or Gateway in the member cluster when it runs in
 // the member cluster.
-// It will clean up stale MemberClusterAnnounce resources in the leader cluster if no corresponding member
-// cluster in the ClusterSet.Spec.Members when it runs in the leader cluster.
+// It will clean up stale MemberClusterAnnounce resources in the leader cluster if the last update timestamp
```

Comment:
"if the MemberClusterAnnounce timestamp annotation has not been updated for memberClusterAnnounceStaleTime (24 hours)"